YouTube.js

```diff
@@ -110,6 +109,18 @@ function scrape(doc, url) {
 	if (description) {
 		item.abstractNote = description;
 	}
+	let tags = ZU.xpathText(doc, '//meta[@name="keywords"]/@content');
+	if (tags) {
+		tags.split(',').forEach(function (tag) {
+			item.tags.push(tag.trim());
+		});
+	}
+	
+	// add video key as signatur
+	let signaturMatches = item.url.match(/.+v=(.+)$/);
```

Comment:
I don't really think we need/want this. It's already in the URL and YouTube isn't actually a library catalog. I don't know of a citation style that would want something like `Zoteron. "Zotero Intro" (2007). YouTube, pq94aBrc0pY.` No one would know what that means!